### ISSUE TITLE
When will this contract be deployed to the mainnet?

### ISSUE THREAD

**Issue by gzliudan** [wyvernprotocol/wyvern-v3#58]
Is v3.1 of this project production ready? And is there a plan to deploy to the main network?

**Comment by eddybounasr** [comment]
which contracts do i have to deploy on mainnet if i would like to ?? in migrations we deploy just Exchange and Registry .. do i need the Atomicizer and WyvernStatic ?

**Comment by cwgoes** [comment]
No, those already exist on the mainnet, so you can just reuse them.

**Comment by eddybounasr** [comment]
are u available for a call discussion about the wyvern protocol ?
